### PR TITLE
AS in peer dependency

### LINKED ISSUE #4: Updating assemblyscript
If your repo has an updated assemblyscript, this plugin is still using the old version.

### PR DESCRIPTION
**Pull request by munrocket**
Fixes: #4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "keywords": [],
   "author": "Surma <surma@surma.dev>",
   "license": "Apache-2.0",
-  "dependencies": {
+  "peerDependencies": {
     "assemblyscript": "^0.9.4"
```
Comment (surma) [inline package.json]:
```suggestion
    "assemblyscript": "*"
```

With major=0 version, every minor version is considered a breaking change in semver, so I am wondering if I should just allow any version for now? WDYT?

**Review by surma — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (munrocket) [reply package.json]: Yeah you right, we need to use `"assemblyscript": ">=0.9.4"`, or `"*"` if it works.

**Comment by surma** [comment]
Published in v1.3.0 :)

**Comment by surma** [comment]
Thanks for your help, @munrocket!

**Comment by munrocket** [comment]
Hooray 🎉

**Pull request merged**